service/src/main/java/bio/terra/externalcreds/controllers/OidcApiController.java

```diff
@@ -50,7 +50,7 @@ public ResponseEntity<List<String>> listProviders() {
   public ResponseEntity<LinkInfo> getLink(Provider providerName) {
     var samUser = samUserFactory.from(request);
     var linkedAccount =
-        linkedAccountService.getLinkedAccount(samUser.getSubjectId(), providerName.name());
+        linkedAccountService.getLinkedAccount(samUser.getSubjectId(), providerName.toString());
```

Comment:
we should probably just pass the entire `Provider` here, and let whatever needs the string value do what it needs to do.

Reply:
That's a good point! I initially did it this way because it's used as a string in a lot of places 😕

Reply:
So I started making this change and it impacts a ton of places in the code. And in some tests we would need to convert the provider name string (stored on linked account objects) to an enum so...I'm not convinced it's worth making this change 🙈. What do you think about leaving it as-is?

Reply:
Sure, lets just leave it for now, but can we make a ticket to migrate to using the enum?

Reply:
Oh I just saw your comment! I'm nearly done with the change but I'm running into an issue with the tests so I'll put it in a separate PR so this isn't blocked.